install.js

```diff
@@ -1,13 +1,18 @@
 "use strict";
 
+const process = require("process");
 const https = require("follow-redirects").https;
 const tar = require("tar");
 const shell = require("shelljs");
-const version = require("./package.json")["purs-version"];
+const version = process.env.PURS_VERSION;
 const platform = { win32: "win64", darwin: "macos" }[process.platform] || "linux64";
+const urlPrefix = "https://github.com/purescript/purescript/releases";
+const url = version !== undefined
+  ? `${urlPrefix}/download/${version}/${platform}.tar.gz`
+  : `${urlPrefix}/latest/download/${platform}.tar.gz`
```

Comment:
i dont know if this is so good, since this means that there's no guarantee of a stable version being installed